lib/beautiful/init.lua

```diff
@@ -98,6 +100,41 @@ local active_font
 --- The current theme path (if any)
 -- @tfield string beautiful.theme_path
 
+--- Font DPI API
+-- @section font_dpi
```

Comment:
This might be a good idea why for fonts the number of pixels per inch of your monitor is different than for non-fonts.

Reply:
Since I fail to parse my own sentence: The docs here could explain the concept for "fonts get a different DPI" a bit here.